Index: uwsgi-1.4.5/uwsgiconfig.py
===================================================================
--- uwsgi-1.4.5.orig/uwsgiconfig.py
+++ uwsgi-1.4.5/uwsgiconfig.py
@@ -1159,10 +1159,7 @@ def build_plugin(path, uc, cflags, ldfla
     except:
         pass
 
-    if uc:
-        plugin_dest = uc.get('plugin_dir') + '/' + name + '_plugin'
-    else:
-        plugin_dest = name + '_plugin'
+    plugin_dest = name + '_plugin'
 
     shared_flag = '-shared'
 
